funcx_endpoint/funcx_endpoint/executors/high_throughput/funcx_manager.py

```diff
@@ -240,7 +241,7 @@ def __init__(
         self.next_worker_q: list[str] = []  # FIFO queue for spinning up workers.
         self.worker_procs: dict[str, subprocess.Popen] = {}
 
-        self.task_status_deltas: dict[str, tuple[float, TaskState]] = {}
+        self.task_status_deltas: dict[str, tuple[float, TaskState, ActorName]] = {}
```

Comment:
Having implemented TaskTransition in `funcx_common`, is this not prime place to use it?